lib/generators/pakyow/app/templates/Gemfile

```diff
@@ -1,11 +1,10 @@
 source 'https://rubygems.org'
-
-# live on the edge and uncomment this line
-# gem 'pakyow', github: 'pakyow/pakyow'
-
-# use the latest released version
+<% local_pakyow = Gem::Specification.sort_by{ |g| [g.name.downcase, g.version] }.group_by{ |g| g.name }.detect{|k,v| k == 'pakyow'} %>
```

Comment:
If possible, let's move this logic into an external method. It would read better IMO:

```ruby
<%
  if generating_locally?
    gem 'pakyow, github: 'pakyow/pakyow'
  else
    gem 'pakyow', '~> 0.10`
  end
```

While we're at it, do you mind replacing `0.10` with a method call that returns the version of Pakyow that's generating the app?